### DISCUSSION TITLE
fastify-zod

### DISCUSSION THREAD

**Discussion by elierotenberg** [colinhacks/zod#735] [Show and tell]
Hi!
I've just created a library, [`fastify-zod`](https://github.com/elierotenberg/fastify-zod) to facilitate using `zod` and `fastify` together, including generating a clean OpenAPI spec using `fastify-swagger`, and use this spec to generate a human-friendly OpenAPI client using the `typescript-fetch` generator.

I would be very happy to get feedback and if you think it's good, add it to the Ecosystem section of the README.

What do you think?

**Comment by elierotenberg** [comment]
Hello! Having used my library in production for a while now, I hope it is mature enough to be publicly listed.

Should I PR the README and docs?

**Reply by JacobWeisenburger** [reply]
![image](https://user-images.githubusercontent.com/31667350/155746764-cf0b1cef-4691-4e71-b8a7-8af0cc845971.png)
